python/tools/build_tx.py

```diff
@@ -26,6 +26,16 @@
 from trezorlib.cli.btc import INPUT_SCRIPTS, OUTPUT_SCRIPTS
 from trezorlib.protobuf import to_dict
 
+SESSION = requests.Session()
+SESSION.headers.update({"User-Agent": "trezorlib"})
+
+BITCOIN_CORE_INPUT_TYPES = {
+    "scripthash": messages.InputScriptType.SPENDADDRESS,
+    "pubkeyhash": messages.InputScriptType.SPENDP2SHWITNESS,
+    "witness_v0_keyhash": messages.InputScriptType.SPENDWITNESS,
+    "witness_v1_taproot": messages.InputScriptType.SPENDP2SHWITNESS,
```

Comment:
Shouldn't this be `SPENDTAPROOT`?

Reply:
Also `SPENDADDRESS` and `SPENDP2SHWITNESS` are mixed up?

Reply:
well :hankey: